cmd/humanInsertion/humanInsertion.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/maf"
+	"log"
+)
+
+func humanInsertion(mafFile string, outBed_ins string, outBed_del string, species_ins string, species_del string) {
+	//initialize variables
+	mafRecords := maf.Read(mafFile) //Read entire mafFile. mafRecords has type Maf
+	var bedList_ins []*bed.Bed      //initialize 2 bed files
+	var bedList_del []*bed.Bed      //1 bed file for ins, 1 bed file for del
+
+	//go through each line
+	for i, _ := range mafRecords { //each i is a block
+		for k, _ := range mafRecords[i].Species { //each k is a line
+
+			//convert maf to bed, start with getting assembly because it is needed to verify species_ins and species_del
+			assembly_del, chrom_del := maf.SrcToAssemblyAndChrom(mafRecords[i].Species[k].Src) //start with species_del, get assembly (e.g. panTro6), chrom (e.g. chrI)
```

Comment:
I'm wondering why we use "_del" here? This variable is holding the assembly and the chrom before we even checked if it was a deletion or insertion, right? This took me a while to figure out because at first it seems like we somehow already knew we were dealing with a deletion based on the variable names, however my interpretation is that we don't really check for "Eline" until line 25. I understand that this variable continually is reset and only used if in fact that there is a deletion, but I still find it kinda confusing to also label insertions as "del" even if we junk them. Could we call it like assembly_k and chrom_k to just make it clear that it is the assembly and chrom being captured as we loop through k? Please help me if I'm thinking about this all wrong.